src/tests/ftest/container/ContainerAsync.py

```diff
@@ -78,7 +80,9 @@ def setUp(self):
         self.hostlist = self.params.get("test_machines",'/run/hosts/*')
         self.hostfile = WriteHostFile.WriteHostFile(self.hostlist, self.workdir)
         print("Host file is: {}".format(self.hostfile))
+        self.hostname = [socket.gethostname()]
 
+        AgentUtils.run_agent(self.basepath, self.hostname, self.hostlist)
```

Comment:
Don't use the host the test is running on a daos node, client or server.
```suggestion
        AgentUtils.run_agent(self.basepath, [], self.hostlist)
```

This is likely the same for many other tests but I won't comment on each and every one.